Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12-alpine
+
+WORKDIR /excalidraw-backend
+
+COPY package.json package-lock.json ./
```

Comment:
Please copy all the files instead of doing it in stages and thus having multiple layers.